app/modules/asset_groups/models.py

```diff
@@ -75,6 +75,10 @@ class AssetGroupSighting(db.Model, HoustonModel):
     )
     detection_attempts = db.Column(db.Integer, default=0, nullable=False)
 
+    def user_is_owner(self, user: User) -> bool:
+        # AssetGroupSighting has no owner, so uses the AssetGroup one
```

Comment:
I don't love these type annotations since we don't have them everywhere but NBD and there are already a few scattered throughout the codebase.